pkg/recipe/sauce.go

```diff
@@ -0,0 +1,85 @@
+package recipe
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/go-yaml/yaml"
+)
+
+type File struct {
+	Checksum string `yaml:"checksum"` // e.g. "sha256:asdjfajdfa" w. default algo
+	Content  []byte `yaml:"-"`
+}
+
+// Sauce represents a rendered recipe
+type Sauce struct {
+	Recipe Recipe          `yaml:",inline"`
+	Values VariableValues  `yaml:"values,omitempty"`
+	Files  map[string]File `yaml:"files"`
+}
+
+const (
+	SauceFileName = "sauce"
```

Comment:
Not sure do we want to have on project directory `sauce.yaml` or `sauces.yaml`

Reply:
Decided to rename it to `sauces.yml`

Reply:
One option is to still have it as `recipe.yml` so end user doesn't need to care about this separation of recipe and sauce